### DISCUSSION TITLE
Validating config error

### DISCUSSION THREAD

**Discussion by jaumexb** [UnchartedBull/OctoDash#2872] [Q&A] [answered]
![16510894261187320512518659697460](https://user-images.githubusercontent.com/63864029/165620813-a63826b1-a04d-4c88-866d-52379346fc1d.jpg)
I have a small error, I have recently updated octodash, and when I turn on and follow all the instructions I get this error.  I have version 1.4 of octoprint and I can't update.

**Comment by jneilliii** [comment, marked as answer]
Probably easiest fix is deleting the config and starting over. SSH to the pi and run these commands.

```
rm /home/pi/.config/octodash/config.json
sudo service getty@tty1 restart
```

**Reply by jaumexb** [reply]
Thank you very much, if it wasn't for you I wouldn't have solved the problem

**Comment by IamPatticus** [comment]
Thanks! That worked for me too!
